### DISCUSSION TITLE
ı try to lower ram used qtile

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3086] [General]
this is default config file 
![2021-12-10-182918_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/145600475-a801ccb4-1958-4647-9f25-20d8e0edb3c6.png)
![2021-12-10-182934_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/145600483-1e342d23-7dfe-425e-8534-f960b70e61f2.png)

this is my config file 
![2021-12-10-182253_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/145600496-a67424f0-b58d-42fb-8986-34fca6652e58.png)
![2021-12-10-182303_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/145600505-d31501cf-a29d-4dd6-b743-592031a8300b.png)

ı need new idea how ı lower ram usage

**Comment by m-col** [comment]
If you're wanting to track down the memory usage a good strategy is disabling a section of the config and seeing effect that section has on memory. This would help you track down e.g. which widget(s) are the culprits. The first thing that comes to mind is the images you have in the bar - the images will be loaded from disk and stored in memory for rendering.
